src/Microsoft.Android.Templates/android-bindinglib/AndroidBinding1.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net9.0-android</TargetFramework>
+    <TargetFramework>net10.0-android</TargetFramework>
```

Comment:
At some point we should auto generate these files so that we don't have to keep updating them.

Reply:
I did this for `AutoImport.props`, but yeah each year we should do a couple. This year had a lot of blockers, though...